### ISSUE TITLE
Unhandled Exception on start

### ISSUE THREAD

**Issue by FattusMannus** [sdesalas/trifleJS#25]
Trying to run TrifleJs from command line give me:

Unhandled Exception: System.IO.FileLoadException: Could not load file or assembl
y 'Noesis.Javascript, Version=0.0.0.0, Culture=neutral, PublicKeyToken=ae36d046c
7f89f85' or one of its dependencies. The application has failed to start because
 its side-by-side configuration is incorrect. Please see the application event l
og or use the command-line sxstrace.exe tool for more detail. (Exception from HR
ESULT: 0x800736B1)
File name: 'Noesis.Javascript, Version=0.0.0.0, Culture=neutral, PublicKeyToken=
ae36d046c7f89f85' ---> System.Runtime.InteropServices.COMException (0x800736B1):
 The application has failed to start because its side-by-side configuration is i
ncorrect. Please see the application event log or use the command-line sxstrace.
exe tool for more detail. (Exception from HRESULT: 0x800736B1)
   at TrifleJS.Program.Interactive()
   at TrifleJS.Program.Main(String[] args)

Event viewer gives me:

Activation context generation failed for "C:\TrifleJS\Noesis.Javascript.dll". Dependent Assembly Microsoft.VC90.CRT,processorArchitecture="x86",publicKeyToken="1fc8b3b9a1e18e3b",type="win32",version="9.0.30729.6161" could not be found. Please use sxstrace.exe for detailed diagnosis.

Is this because I am running on 64 bit? I should have all runtimes etc installed.

**Comment by sdesalas** [comment]
I added a new version of [TrifleJS.Latest.zip](https://github.com/sdesalas/trifleJS/raw/master/Build/Binary/TrifleJS.Latest.zip) with bundled dependencies. Turns out I only had to add the manifest that came with Noesis.Javascript.dll.. I dont have a computer handy for testing but this might have made the issue go away.

**Comment by jouni-kantola** [comment]
*TL;DR*
Reading up in this issue I downloaded ```Noesis.Javascript v0.7 - Binaries.zip (http://javascriptdotnet.codeplex.com/releases/view/88504)``` and referenced the v4.0 binaries in the project and rebuild, and that worked. I'm guessing a beta for .NET Framework 4+ would be of good use, I'm guessing most people use it.

*Issue with TrifleJS dists:*
I cannot get the executable going. It doesn't matter wheter i run TrifleJs.exe from stable v4.0 or latest beta v0.5. This is the stack trace I get (I hope you can decipher it, even though it's in Swedish):
```
C:\temp\trifle\TrifleJS.Latest
λ TrifleJS.exe examples\phantomwebintro.js

Ohanterat undantag: System.IO.FileLoadException: Det går inte att läsa in filen eller sammansättningen Noesis.Javascript, Version=0.0.0.0, Culture=neutral, PublicKeyToken=ae36d046c7f89f85 eller ett av dess beroenden. Det går inte att starta programmet eftersom dess sida vid sida-konfiguration är felaktig. Kontrollera händelseloggen eller använd kommandoradsverktyget sxstrace.exe för mer information. (Undantag från HRESULT: 0x800736B1)
Filnamn: Noesis.Javascript, Version=0.0.0.0, Culture=neutral, PublicKeyToken=ae36d046c7f89f85 ---> System.Runtime.InteropServices.COMException (0x800736B1): Det går inte att starta programmet eftersom dess sida vid sida-konfiguration är felaktig. Kontrollera händelseloggen eller använd kommandoradsverktyget sxstrace.exe för mer information. (Undantag från HRESULT: 0x800736B1)
   vid TrifleJS.Program.Open(String filename)
   vid TrifleJS.Program.Main(String[] args)
```
A quick translation would be FileLoadException when loading Noesis.Javascript or one of it's dependencies.

So if I instead compile the source with VS2013 targeted to .NET framework 4.5, I get this:
```
C:\temp\trifle\src\trifleJS\bin\Debug (master)
λ TrifleJS.exe examples\phantomwebintro.js
TrifleJS.exe examples\phantomwebintro.js
Setting Version to IE9

Unhandled Exception: System.IO.FileLoadException: Could not load file or assembly 'Noesis.Javascript.dll' or one of its dependencies. Det går inte att starta programmet eftersom dess sida vid sida-konfiguration är felaktig. Kontrollera händelseloggen eller använd kommandoradsverktyget sxstrace.exe för mer information. (Exception from HRESULT: 0x800736B1)
   at TrifleJS.Program.Open(String filename)
   at TrifleJS.Program.Main(String[] args) in c:\temp\trifle\src\trifleJS\Program.cs:line 201
```

So I downloaded the Noesis.Javascript DLL's from Codeplex (http://javascriptdotnet.codeplex.com/releases/view/88504) as @feuyeux suggested. I referenced the dependienced, rebuild, and that worked fine.
```
C:\temp\trifle\src\trifleJS\bin\Debug (master)
λ TrifleJS.exe examples\phantomwebintro.js
TrifleJS.exe examples\phantomwebintro.js
Setting Version to IE9
new FileSystem()
new System()
new ChildProcess()
WebPage.fireEvent('navigationrequested', uuid, args)
WebPage.fireEvent('loadstarted', uuid, args)
...
```

**Comment by elvarb** [comment]
I was trying to get it to run in Windows Server 2012 R2 and got this error while using TrifleJS.Latest.zip, by adding the Noesis.Javascript.dll from the Stable release to the directory the error I get is

phantom.fireEvent() (1,0): ReferenceError: phantom is not defined
